spec/functional/bootstrap_download_spec.rb

```diff
@@ -100,6 +100,14 @@
         run_download_scenario
       end
     end
+
+     context "when provided a custom msi_url with space in path to fetch from" do
+      let(:mock_bootstrap_context) { Chef::Knife::Core::WindowsBootstrapContext.new(
+        { :msi_url => "file:///C:/Program Files/Windows NT/Accessories/wordpad.exe" }, nil, { :knife => {} }) }
+      it "downloads the chef-client MSI from a custom path with spaces during winrm bootstrap" do
```

Comment:
Can you add a test for the other direction of this - does it still work if I do file://C:/Program%20Files ... ?